deployments/helm/vpn/templates/vppagent-firewall-nse.tpl

```diff
@@ -15,9 +15,11 @@ spec:
     spec:
       containers:
         - name: firewall-nse
-          image: {{ .Values.registry }}/networkservicemesh/vppagent-firewall-nse:{{ .Values.tag }}
```

Comment:
Do we need to pass a command for this container?

Reply:
No, application is selected using the `VPP_APP` environment variable.